aemedge/blocks/popular-blogs/popular-blogs.css

```diff
@@ -1,12 +1,181 @@
-.section.popular-blogs-container
-{
-    background: #f2f2f2;
+.section.popular-blogs-container {
     margin: 0 auto;
     width: 100%;
     padding: 37px 0;
+    background: #f2f2f2;
+    margin-bottom: 50px;
+}
+
+.popular-blogs .slides-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.popular-blogs .slides-wrapper {
+    position: relative;
+}
+
+.popular-blogs .heading-wrapper {
+    display: flex;
+    font-size: 42px;
+    justify-content: center;
+    flex-flow: wrap column;
+    align-items: center;
+}
+
+.popular-blogs .heading-wrapper img {
+    max-height: 36px;
+    max-width: 36px;
+    vertical-align: middle;
 }
 
-.popular-blogs{
+.popular-blogs .headline {
+    color: #1d1d1d;
+    font-size: 20px;
+    font-weight: bold;
+    line-height: 30px;
+    margin: 7px auto 44px;
+    max-height: 30px;
+    text-align: center;
+}
+
+.popular-blog-title {
+    color: #000a12;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 26px;
+    max-width: 288px;
+    max-height: 164px;
+}
+
+.popular-blogs .blog-slides {
     display: flex;
+    scroll-behavior: smooth;
+    scroll-snap-type: x mandatory;
+    overflow: scroll clip;
```

Comment:
remove scroll clip so the scrollbar shows

Reply:
removing the _overflow: scroll_ clip is making the arrows disappear and the scroll bar is appearing on mobile , so keeping it there

Reply:
Ok, just have the overflow: scroll clip available on mobile size.